client/src/components/github.js

```diff
@@ -0,0 +1,35 @@
+const axios = require("axios");
```

Comment:
IMO this file should be in the backend rather than the frontend doing it, and to add, I believe this should actually be a part of scholly - which is in my branch

Reply:
Oh right, this file is created by @kdav108  in the Logiciel branch right before the PI2 demo. Shall I remove this file?

Reply:
I think leave it for now in case we have some discussion over it in my PR / meetings, and then we can delete it before you merge